package.json

```diff
@@ -589,43 +589,46 @@
     {
       "id": "Razor",
       "description": "Razor Language Server (Windows / x64)",
-      "url": "https://download.visualstudio.microsoft.com/download/pr/badc9874-13de-4e0c-bad3-ea51383f5a51/477b855ceecec09efb2045364090418d/razorlanguageserver-win-x64-7.0.0-preview.23275.2.zip",
+      "url": "https://download.visualstudio.microsoft.com/download/pr/51d2bf3c-cb3c-4385-b59f-d4d6c9a96743/3b7fc7c2f06e48d78bec2125adcc41aa/razorlanguageserver-win-x64-7.0.0-preview.23328.2.zip",
```

Comment:
nvm, I see the guid changes.  But maybe we could just specify the guid and the version number in the launch.json...
Anyway, no need to do now just thinking about later improvements.

Reply:
There are actually two guids here, one is for the folder which is the same for each insertion, but one is unique per file, so we would still need some part of each url.

Reply:
Having said that, as discussed previously, if this isn't going to be a runtime dependency, we can upload it to the same blob that omnisharp goes to if that makes it easier.